src/braket/ahs/atom_arrangement.py

```diff
@@ -41,18 +43,21 @@ def __init__(self):
         self._sites = []
 
     def add(
-        self, coord: Tuple[Number, Number], site_type: SiteType = SiteType.FILLED
+        self,
+        coord: Union[Tuple[Number, Number], np.ndarray],
+        site_type: SiteType = SiteType.FILLED
     ) -> AtomArrangement:
         """Add a coordinate to the atom arrangement.
 
         Args:
-            coord (Tuple[Number, Number]): The coordinate of the atom (in meters). The coordinates
-                can be int, float or Decimal.
+            coord (Union[Tuple[Number, Number], np.ndarray]): The coordinate of the
+                atom (in meters). The coordinates can be a tuple of int, float,
+                Decimal, or a numpy array of shape (2,)
```

Comment:
Slight ambiguity in that the tuple can be interpreted to contain two numpy arrays
```suggestion
                atom (in meters). The coordinates can be a numpy array of shape (2,)
                or a tuple of int, float, Decimal
```